### PR TITLE
Update the markup of the footers

### PR DESCRIPTION
**Pull request by carolinan**
**Description**
Updates the markup of the footers.

I have used the markup from #130, with these exceptions:
- I re-added the aria labels to the navigation blocks
- I replaced the x-small font size with small, since x-small has been removed.
- I replaced the 20px spacers with the same size, but using the spacing preset.
- I replaced two background colors by applying the section styles, and removed hard coded colors.

Newsletter footer: Replace the site title with a heading with a single word
Footer with columns: Make it wide inside a full width group.

Closes
https://github.com/WordPress/twentytwentyfive/issues/130

**Testing Instructions**

Apply the PR
View the footers in the editor and the front, on different screen widths.

**Screenshots**

### CODE REVIEW

**Comment by jasmussen** [comment]
Looks good. A few things that may or may not be part of this PR, the registered trademark symbol makes this word wrap:

Screenshot 2024-09-03 at 10 07 23

Regardless of what we may or may not replace that with, we probably need a slightly smaller font size there.

Bea mentioned these should be XL. They are XL in trunk, but appear to be Large here. 

Screenshot 2024-09-03 at 10 07 38

There's something going on with the font sizes, especially for the navigation, in this one:

Screenshot 2024-09-03 at 10 08 01

**Comment by carolinan** [comment]
>There's something going on with the font sizes, especially for the navigation, in this one

I am not able to reproduce this. The heading above, and the navigation are set to use the medium size.
Individual custom links can have their font size changed on the block, but by default new items added to the navigation should all be medium.


----
I found that when one navigation block has more items, they don't align. In the wrapping row block, the vertical alignment is set to center. It should probably be aligned to top, what do you think?
image

**Comment by beafialho** [comment]
> There's something going on with the font sizes, especially for the navigation, in this one:

@jasmussen I suspect the screenshot you shared is from a pattern from the pattern directory. The ones from TT5 are looking good to me.

**Comment by beafialho** [comment]
Thank you @carolinan, I left a comment above but other than that, they're looking good on my end.

File: patterns/footer-columns.php [hunk 1]
```diff
@@ -12,26 +12,27 @@
...
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+	<div class="wp-block-group alignwide">
 		<!-- wp:group {"align":"full","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between","verticalAlignment":"top"}} -->
 		<div class="wp-block-group alignfull">
 			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20","padding":{"top":"0","bottom":"0","left":"0","right":"0"}}},"layout":{"type":"constrained"}} -->
 			<div class="wp-block-group" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
-				<!-- wp:site-title {"level":2,"fontSize":"xx-large"} /-->
+				<!-- wp:site-title {"style":{"typography":{"fontSize":"70px"}}} /-->
```
Comment (carolinan) [inline patterns/footer-columns.php]: This must not be an H1, which is the default level for the site title.
And the custom CSS variable should be used for the 70px.

**Comment by carolinan** [comment]
> Thank you @carolinan, I left a comment above but other than that, they're looking good on my end.

I'm sorry, which comment, the one about the core block?

**Comment by beafialho** [comment]
> > Thank you @carolinan, I left a comment above but other than that, they're looking good on my end.
> 
> I'm sorry, which comment, the one about the core block?

This one:

For this one big heading, let's make a small change to better match the design:

Font size: **26vw**
Letterspacing: **-0.02em**
Appearance: **Semi Bold**

**Review by juanfra — COMMENTED** [review]
It is working well to me, using the styles that are coming from the GH issue. 

I noticed the font weight on the "Stories" text for the "Footer with newsletter signup", but it looks like Bea already highlighted that.

**Comment by carolinan** [comment]
26vw looks like this on my screen at 1920x1080.
Firefox:
newsletter-footer-ff
Chrome:
newsletter-footer-chrome

**Comment by beafialho** [comment]
> 26vw looks like this on my screen at 1920x1080. Firefox: newsletter-footer-ff Chrome: newsletter-footer-chrome

Oh, that doesn't look good. Please keep the **18vw** we had before 👍

**Pull request merged**